Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -2807,4 +2813,55 @@ defineSuite([
         });
     });
 
+    it('clipping planes cull hidden tiles', function() {
```

Comment:
Instead of / or in addition to these tests, can you add tests like:
* Calls `update` and checks that `n` draw commands are returned, sets a clipping plane, and checks that `m` draw calls are returned under different cases, e.g., when all draw commands are culled, no draw commands are culled, when a subset are culled in an HLOD fashion
* Include `i3dm` in a tileset
* Renders, checks that a pixel was colored, sets a clip plane to clip everything, renders, then checks that nothing was rendered
* Multiple planes with and without combine

Similiar comment for terrain.

Also, looks like explicit tests are missing for `Model`?

The tests in PointCloud3DTileContentSpec.js look pretty solid - why not apply the pattern throughout?  Don't worry about the duplication or encapsulate it.

Finally, please check code coverage for the new files and make it as high as reasonable.